### PR TITLE
Move to utils

### PR DESCRIPTION
**Pull request by rachel-fenichel**
### Resolves

Merge stuff

### Proposed Changes

Move two functions: block `isRecyclable` and `duplicateAndDragCallback` into the scratch blocks utils file.

### Reason for Changes

This file is where we put things that are package and definitely not going into blockly, to reduce merge conflicts.

### CODE REVIEW

File: core/scratch_blocks_utils.js [hunk 1]
```diff
@@ -114,3 +114,142 @@ Blockly.scratchBlocksUtils.compareStrings = function(str1, str2) {
...
+ * option in Scratch Blocks.  The block is duplicated and attached to the mouse,
+ * which acts as though it were pressed and mid-drag.  Clicking the mouse
+ * releases the new dragging block.
+ * @param {!Blockly.BlockSvg} oldBlock The block that will be duplicated.
+ * @return {Function} A callback function that duplicates the block and starts a
+ *     drag.
+ * @package
+ */
+Blockly.scratchBlocksUtils.duplicateAndDragCallback = function(oldBlock) {
+  var oldBlock = this;
```
Comment (kchadha) [inline core/scratch_blocks_utils.js]: This line should no longer be here.

**Review by kchadha — CHANGES_REQUESTED** [review]
Copy-paste/refactor error noted in the comments.

**Comment by rachel-fenichel** [comment]
Good catch.  Fixed.

**Review by kchadha — APPROVED** [review]
LG

**Pull request merged**